lib/sequelize.js

```diff
@@ -854,14 +854,16 @@ Sequelize.prototype.sync = function(options) {
       return Promise.reject('Database does not match sync match parameter');
     }
   }
-  var when;
-  if (options.force) {
-    when = this.drop(options);
-  } else {
-    when = Promise.resolve();
-  }
 
-  return when.then(function() {
+  return Promise.try(function () {
+    if (options.hooks) {
+      return self.runHooks('beforeBulkSync', options);
+    }
```

Comment:
What's the default value of `options.hooks`? `undefined`? should probably be `true`